drivers/gles3/rasterizer_scene_gles3.cpp

```diff
@@ -3,6 +3,11 @@
 #include "os/os.h"
 #include "rasterizer_canvas_gles3.h"
 
+#ifdef IPHONE_ENABLED
+// for some reason glClearDepth seems to have been removed in iOS ES3.h
+#define glClearDepth glClearDepthf
```

Comment:
I think OpenGL 3.3 has glClearDepthf , so it might be safe to just use that function